6-Typescript/server/tests/it/create-todo.test.ts

```diff
@@ -1,31 +1,34 @@
-import { createTodo, createTodoWithMissingChecked } from './../env/random';
-import { TestKit } from '../env/TestKit';
-import { ITodo } from '../../interfaces/ITodo';
+import { createTodo, createTodoWithMissingChecked } from "../env/mockData";
+import { TestKit } from "../env/TestKit";
+import { ITodo } from "../../interfaces/interfaces";
 describe("POST new todo /api/todos ", () => {
-    const testKit = new TestKit()
+    const testKit = new TestKit();
     testKit.beforeAndAfter();
 
     describe("given new todo to create ", () => {
-        it("should return 201 and that todo is equal to the one we inserted", async () => {            
-            const todo: ITodo = createTodo()
-            testKit.appDriver.setUserId(todo.userId)
-            const response = await testKit.appDriver.addTodo(todo)
-            const checkTodoInDb: ITodo | null = await testKit.mongoDriver.modelGetTodoById(todo.userId ,todo.todoId)
+        it("should return 201 and that todo is equal to the one we inserted", async () => {
+            const todo: ITodo = createTodo();
+            testKit.appDriver.setUserId(todo.userId);
+            const response = await testKit.appDriver.addTodo(todo);
+            const checkTodoInDb: ITodo | null =
+                await testKit.mongoDriver.modelGetTodoById(
+                    todo.userId,
+                    todo.todoId
+                );
             expect(checkTodoInDb).toEqual(todo);
             expect(response.status).toBe(201);
-        })
-    })
-
+        });
+    });
 
     describe("given todo with missing value in payload ", () => {
-        it("should return 400", async () => {            
-            const todo:Partial<ITodo> = createTodoWithMissingChecked()
-            testKit.appDriver.setUserId(todo.userId)
+        it("should return 400", async () => {
+            const todo: Partial<ITodo> = createTodoWithMissingChecked();
+            testKit.appDriver.setUserId(todo.userId);
             try {
-                const response = await testKit.appDriver.addTodo(todo)
-            } catch(err: any) {                
+                const response = await testKit.appDriver.addTodo(todo);
+            } catch (err: any) {
```

Comment:
`any`

Reply:
again - this is the error i got when i tried to use a diff type name:
'Catch clause variable type annotation must be 'any' or 'unknown' if specified'